www/layout.html

```diff
@@ -87,7 +103,71 @@
 
     <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
     <script src="vendor/bootstrap/js/ie10-viewport-bug-workaround.js"></script>
-
+    {% if page_name == 'manage' : %}
```

Comment:
You could also move this code to the `manage.html` file and if you really want it to be organized, you can add another `{% block %}` for the scripts in this layout. That way we don't have to modify `layout.html` everytime we want to add more scripts to specific templates.

Reply:
Cool! Thanks for the tip. Will look into it.